src/App.js

```diff
@@ -1,13 +1,56 @@
 import React from 'react';
-
 import './App.scss';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodList } from './GoodList';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  selectAll = () => {
+    getAll()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  select5Goods = () => {
+    get5First()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  selectRedGoods = () => {
+    getRedGoods()
+      .then((goods) => {
+        this.setState({ goods });
+      });
```

Comment:
You could make just 1 function instead of these 3 and accept callback as a parameter